src/orders/views/OrderDetails/OrderNormalDetails/index.tsx

```diff
@@ -457,6 +459,13 @@ export const OrderNormalDetails: React.FC<OrderNormalDetailsProps> = ({
           })
         }
       />
+      <OrderRefundDialog
+        open={params.action === "add-refund"}
+        onClose={closeModal}
+        // TODO: Add redirect to refund view
+        onConfirm={() => null}
```

Comment:
question: do you have any reason why didn't implemented it now

Reply:
Yes - refund view is not implemented (it's going to be done next in merx-55). We want to avoid big PR-s.